### PR TITLE
Make GKE-CI test firewalls work on Jenkins

### PR DESCRIPTION
**Pull request by cjcullen**

### CODE REVIEW

File: cluster/gke/util.sh [hunk 1]
```diff
@@ -169,7 +169,7 @@ function test-setup() {
   detect-minions >&2
 
   # At this point, CLUSTER_NAME should have been used, so its value is final.
-  MINION_TAG=$($GCLOUD compute instances describe ${MINION_NAMES[0]} | grep -o "gke-${CLUSTER_NAME}-.\{8\}-node" | head -1)
+  MINION_TAG=$($GCLOUD compute instances describe ${MINION_NAMES[0]} --project="${PROJECT}" --zone="${ZONE}" | grep -o "gke-${CLUSTER_NAME}-.\{8\}-node" | head -1)
```
Comment (mbforbes) [inline cluster/gke/util.sh]: Please add ZONE to the "Assumed vars:" list above

File: cluster/gke/util.sh [hunk 2]
```diff
@@ -311,7 +311,7 @@ function test-teardown() {
   detect-project >&2
   detect-minions >&2
   # At this point, CLUSTER_NAME should have been used, so its value is final.
-  MINION_TAG=$($GCLOUD compute instances describe ${MINION_NAMES[0]} | grep -o "gke-${CLUSTER_NAME}-.\{8\}-node" | head -1)
+  MINION_TAG=$($GCLOUD compute instances describe ${MINION_NAMES[0]} --project="${PROJECT}" --zone="${ZONE}" | grep -o "gke-${CLUSTER_NAME}-.\{8\}-node" | head -1)
```
Comment (mbforbes) [inline cluster/gke/util.sh]: Please add ZONE to the "Assumed vars:" above

**Comment by mbforbes** [comment]
Two nits. If the calls work when you run them one-off (I know the positioning of `--project` and `--zone` are finicky in some `gcloud` command groups), then this is LGTM mod the nits.

File: cluster/gke/util.sh (hunk 2 above)
Comment (cjcullen) [inline cluster/gke/util.sh]: Done.

File: cluster/gke/util.sh (hunk 1 above)
Comment (cjcullen) [inline cluster/gke/util.sh]: Done.

**Comment by cjcullen** [comment]
This order appears to work. I think the 'gcloud compute' commands are fine either way.

File: cluster/gke/util.sh (hunk 1 above)
Comment (mbforbes) [inline cluster/gke/util.sh]: You actually don't want PROJECT in that list because detect-project sets it :-)

File: cluster/gke/util.sh (hunk 2 above)
Comment (mbforbes) [inline cluster/gke/util.sh]: Same here

File: cluster/gke/util.sh (hunk 1 above)
Comment (cjcullen) [inline cluster/gke/util.sh]: Fiiine. Done.

File: cluster/gke/util.sh (hunk 2 above)
Comment (cjcullen) [inline cluster/gke/util.sh]: And done.

**Comment by mbforbes** [comment]
LGTM!

**Pull request merged**